### PR TITLE
fix kube-dns

### PR DESCRIPTION
**Pull request by rchirakk**
use api-server port from configuration
fixes kube-dns

Signed-off-by: Ranjith <rchirakk@users.noreply.github.com>

### CODE REVIEW

File: mgmtfn/k8splugin/kubeClient.go [hunk 1]
```diff
@@ -296,9 +307,16 @@ func (c *APIClient) WatchServices(respCh chan SvcWatchResp) {
 			for _, port := range wss.Object.Spec.Ports {
 				ps := core.PortSpec{Protocol: string(port.Protocol),
 					SvcPort:  uint16(port.Port),
-					ProvPort: uint16(port.TargetPort),
 					NodePort: uint16(port.NodePort),
 				}
+
+				// handle 'kubernetes' service
```
Comment (gkvijay) [inline mgmtfn/k8splugin/kubeClient.go]: Can you add some comments to indicate that this is a temporary fix

Comment (rchirakk) [inline mgmtfn/k8splugin/kubeClient.go]: added

**Comment by rchirakk** [comment]
@gkvijay PTAL

**Comment by DivyaVavili** [comment]
LGTM

**Pull request merged**